app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = [Person(person_data["name"], person_data["age"])
+                   for person_data in people]
+
+    for person_data in people:
+        person = Person.people.get(person_data["name"])
+        if person_data.get("wife") and person_data["wife"]:
+            person.wife = Person.people.get(person_data["wife"])
+        elif person_data.get("husband") and person_data["husband"]:
+            person.husband = Person.people.get(person_data["husband"])
```

Comment:
The logic for setting the `wife` and `husband` attributes is correct, but it could be simplified. Since you're already using `dict.get()`, you don't need to check `person_data['wife']` or `person_data['husband']` again. You can directly assign the result of `Person.people.get()` to `person.wife` or `person.husband`, as `dict.get()` will return `None` if the key doesn't exist, which is a valid assignment.